frontend/package.json

```diff
@@ -36,7 +36,7 @@
     "shortid": "^2.2.15"
   },
   "scripts": {
-    "start": "SET REACT_APP_API_BASE_URL=http://softservedemo.herokuapp.com/ & react-scripts start",
+    "start": "react-scripts start",
```

Comment:
use correct package.json, it is not correct

Reply:
done